sprokit/tests/pipelines/test_motion.pipe

```diff
@@ -0,0 +1,51 @@
+# ================================================================
+#
+# Using a pedestrian data set from Kaggle:
+#
+#   https://www.kaggle.com/smeschke/pedestrian-dataset
+#
+# Video is 30fps
+#
+process input
+  :: video_input
+  video_filename      = /Data/Kaggle/smeschke/pedestrian-dataset/crosswalk.avi
+  frame_time         = .3333
+  video_reader:type  = vidl_ffmpeg
+
+# ================================================================
+#
+# We can add a config block for two types of motion detectors
+# Only the one we select gets used
+#
+process motion
+  :: detect_motion
+
+  block algo
+    #type =  detect_motion_mog2
+    type =  detect_motion_3frame_differencing
```

Comment:
I don't think this configuration will work as it is. The block name has to match the algorithm type string.

Reply:
Fixed the pipeline names